cfgov/unprocessed/js/apps/owning-a-home/ExpandableGroup.js

```diff
@@ -0,0 +1,67 @@
+// Required modules.
```

Comment:
Is there no way to use either the CF expandables code or the existing cfgov-refresh expandables code? Three different versions seems like we're letting this pattern get away from us.

Reply:
The OAH expendables are somewhat different that the version that already existed in this repo. It's really just a conversion of the old version to remove jQjuery. After OAH goes in, I will work on unifying all three versions.

Reply:
Awesome, thank you. I've been wanting to tackle that in cfgov but haven't found the time to work on it.